blog/2021-05-25-sensor_attributes.md

```diff
@@ -0,0 +1,25 @@
+---
+author: Erik Montnémery
+authorURL: https://github.com/emontnemery
+title: New sensor properties
```

Comment:
Maybe add some more context to the title.
e.g., `New sensor properties for long-term statistics`